lib/model/nw/schema/ComplexType.js

```diff
@@ -57,12 +60,34 @@ class ComplexType extends aggregate(AnnotationTarget, SharedComplexType) {
    * @memberof ComplexType
    */
   getProperty(name) {
-    let prop = this.properties.find((p) => p.name === name);
-    if (!prop) {
+    let property;
+    let properties;
+
+    const propertyNameParts = name.split("/");
+    const propertyName = propertyNameParts[propertyNameParts.length - 1];
+    const navigationPropertyName = propertyNameParts[0];
```

Comment:
It would be better to implement nesting of navigation properties straight away. It won't require much more effort probably.

Reply:
It is just about finding definition of property by "path", because annotation on EntityType can be routed to Navigation Property (there are annotations defined by path `navigationPropertyName/columnName`). I know it is strange, but it is current implementation in Netweaver metadata.

Reply:
I mean nested navigation property target path like `navProp1/navProp2/prop`

Reply:
It can be second step.